Datadog.Trace.Minimal.sln

```diff
@@ -50,6 +50,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Datadog.Core.Tools", "tools
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Datadog.Trace.AspNet", "src\Datadog.Trace.AspNet\Datadog.Trace.AspNet.csproj", "{B34EDBC7-C5FB-409D-8472-BC7469D6F2BD}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Datadog.Trace.Build", "src\Datadog.Trace.Build\Datadog.Trace.Build.csproj", "{ADE25B77-2608-4E3E-96B0-6144A6466F84}"
```

Comment:
Can we call this `Datadog.Trace.MSBuild`? Unless the goal is to add more build-related stuff in addition to msbuild.

Reply:
Sure, I’ll change it

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/811/commits/3d3c77738cbe08fec04bc7c00fbeb442e1ad6ebd